src/sentry/static/sentry/app/components/events/interfaces/richHttpContent.jsx

```diff
@@ -32,7 +32,13 @@ const RichHttpContent = React.createClass({
             : [[k, val]] // key has single value
         );
       }, [])
-      .sort(function([keyA], [keyB]) {
+      .sort(function([keyA, valA], [keyB, valB]) {
+        // if keys are identical, sort on value
+        if (keyA === keyB) {
+          keyA = valA;
+          keyB = valB;
```

Comment:
yeah it's not a big deal. I just don't like assigning to arguments, even though it's probably ok here